lhotse/bin/lhotse

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+Use this script like:
+
+$ lhotse --help
+$ lhotse make-feats --help
+$ lhotse make-feats --compressed audio_manifest.yml mfcc_dir
+
+$ lhotse default-feature-config feat-conf.yml
+"""
+from pathlib import Path
+from typing import Optional
+
+import click
+
+from lhotse.audio import AudioSet
+from lhotse.features import FeatureExtractor, FeatureSetBuilder
+from lhotse.supervision import SupervisionSet
+from lhotse.utils import Pathlike
+
+
+@click.group()
+def cli():
+    """
+    The shell entry point to Lhotse, a tool and a library for audio data manipulation in high altitudes.
+    """
+    pass
+
+
+@cli.command()
+@click.argument('output_config', type=click.Path())
+def default_feature_config(output_config):
+    """Save a default feature extraction config to OUTPUT_CONFIG."""
+    FeatureExtractor().to_yaml(output_config)
+
+
+@cli.command()
+@click.argument('audio_manifest', type=click.Path(exists=True, dir_okay=False))
+@click.argument('output_dir', type=click.Path())
+@click.option('-s', '--segmentation-manifest', type=click.Path(exists=True, dir_okay=False),
+              help='Optional manifest specifying the regions, for which features are to be extracted. '
+                   'When not specified, features will be extracted for the entire recording. '
+                   'Supervision manifest can be used here.')
+@click.option('-a', '--augmentation-manifest', type=click.Path(exists=True, dir_okay=False),
+              help='Optional manifest specifying augmentation transforms that can be applied to recordings.')
+@click.option('-f', '--feature-manifest', type=click.Path(exists=True, dir_okay=False),
+              help='Optional manifest specifying feature extractor configuration.')
+@click.option('--compressed/--not-compressed', default=True, help='Enable/disable lilcom for features compression.')
+@click.option('-t', '--lilcom-tick-power', type=int, default=-8,
+              help='Determines the compression accuracy; '
+                   'the input will be compressed to integer multiples of 2^tick_power')
+@click.option('-j', '--num-jobs', type=int, default=1, help='Number of parallel processes.')
```

Comment:
Do we have a mechanism for splitting up manifest files into pieces?  That could be another way of splitting it into jobs, in case we want them to be on multiple machines (e.g. a queue).

Reply:
I'll add it in a follow-up PR as two CLI modes: `lhotse split` and `lhotse combine`